### PR TITLE
feat: add icon component #1221

### PR DESCRIPTION
**Pull request by rast-mk1**

### CODE REVIEW

File: app/components/Icon/Icon.stories.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+export const Spin = Template.bind({});
+
+Spin.args = {
+  spin: true,
+};
+export const Pulse = Template.bind({});
+
+Pulse.args = {
+  pulse: true,
+};
```
Comment (weronika299) [inline app/components/Icon/Icon.stories.js:40]: I don't fully understand what is characteristic of pulse and spin now. Will styles be added to them as well?

Comment (rast-mk1) [reply app/components/Icon/Icon.stories.js:40]: Both are optional, simple FontAwesome built in features. Spin makes Icon rotate in a smooth manner, while pulse makes it rotate in a more rough way. Custom css can be used instead.

Comment (weronika299) [reply app/components/Icon/Icon.stories.js:40]: Ok, I get it. :)

**Review by weronika299 — APPROVED** [review]

**Review by MS151994 — APPROVED** [review]
looks good to me

**Review by wlechowicz — APPROVED** [review]

File: app/components/Icon/Icon.stories.js [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+
+BlackBackground.args = {
+  backgroundColor: 'black',
+  color: 'white',
+};
+
+export const Spin = Template.bind({});
+
+Spin.args = {
+  backgroundColor: 'white',
```
Comment (marcelina-hasiak) [inline app/components/Icon/Icon.stories.js:159]: #159 #166 Just quick request for simple change: could you please set background-color to color, which won't be white? Icon animation is invisible in Storybook now
Zrzut ekranu 2022-02-24 o 01 56 51

**Review by marcelina-hasiak — CHANGES_REQUESTED** [review]

File: app/components/Icon/Icon.stories.js (hunk 2 above)
Comment (rast-mk1) [reply app/components/Icon/Icon.stories.js:159]: That's weird, I have no problems like that neither on my localhost nor on PR's storybook preview

Comment (rast-mk1) [reply app/components/Icon/Icon.stories.js:159]: obraz

**Review by marcelina-hasiak — APPROVED** [review]

**Comment by wlechowicz** [comment]
@rast-mk1 check the changes to Icon in Chromatic, if the padding change is intended then approve, if it's not then fix. After everything is approved in Chromatic, you can merge.

**Pull request merged**